Changelog.md

```diff
@@ -2,6 +2,11 @@ OpenCore Changelog
 ==================
 #### v0.8.1
 - Improved `ExtendBTFeatureFlags` quirk on newer macOS versions, thx @lvs1974
+- ocvalidate: always print version compatibility info
+- Source code: codestyle fixes, cleanup and conversions
+- Added notes about DMAR table and `ForceAquantiaEthernet`, thx @kokowski
```

Comment:
Mhm, this could be worth mentioning indeed, though initially I thought it would be ok to just update the docs as users will see it from the latex diff. Please keep this.